cogs/currency.py

```diff
@@ -6,28 +6,59 @@
 class Currency(commands.Cog):
     def __init__(self, bot):
         self.bot = bot
-        self.c = sqlite3.connect('currency.db')
-        self.connect = self.c.cursor()
+        self.conn = sqlite3.connect('currency.db')
+        self.c = self.conn.cursor()
+
+    def modify_balance(self, author, target, amount):
+        self.c.execute("UPDATE CURRENCY SET balance = balance + ? where UID = ?", [amount, author])
+        self.c.execute(
+            "UPDATE CURRENCY SET balance = balance + ? where UID = ?", [-amount, target])
```

Comment:
This also still looks incorrect from the -amount. I'll accept the changes, and adjust it myself.